Plugin/Source/GUI/ModulatableSlider.h

```diff
@@ -23,6 +23,8 @@ class ModulatableSlider : public foleys::AutoOrientationSlider,
     std::unique_ptr<juce::SliderParameterAttachment> attachment;
     chowdsp::FloatParameter* modParameter = nullptr;
     PluginEditorCallback pluginEditorCallback = nullptr;
+    
+    double modulatedValue;
```

Comment:
So it seems like `modulatedValue` is getting written to in a couple spots, but is never being read from?

Reply:
I believe its read `
void ModulatableSlider::timerCallback()
{
    const auto newModulatedValue = modParameter->getCurrentValue();
    if (std::abs (modulatedValue - newModulatedValue) < 0.01)
        return;

    modulatedValue = newModulatedValue;
    repaint();
}`

i the absolute value to see if the difference is big enough, maybe theres a more streamlined way to achieve that